hkm/templates/hkm/snippets/navigation.html

```diff
@@ -18,19 +18,35 @@
 
 	<div id="navigation-toggle" class="navigation-links collapse navbar-collapse">
 		<ul class="nav navbar-nav">
+			{% if not user.profile.is_museum %}
 			<span class="nav-search-form__desktop">
 				<form action="/search/" id="nav-search" class="form-inline ">
 					<input class="nav-search__field" placeholder="{% trans 'Search' %}" type="text" name="search" {% if search_term %} value="{{ search_term }}"{% endif %} />
 					<button class="btn btn-primary nav-search__btn" type="submit"><img src="/static/hkm/img/suurennuslasi.png"></img></button>
 				</form>
-			</span>
-
-			<li class="nav-search-link__mobile"><a href="{% url 'hkm_search' %}">{% trans 'Search' %}</a><li> 
+			</span><li class="nav-search-link__mobile"><a href="{% url 'hkm_search' %}">{% trans 'Search' %}</a><li>
+			{% endif %}
+			{% if user.profile.is_museum %}
```

Comment:
I would merge this and the previous if statement into an if else statement because they seem to be logical opposites.